R/pkg/inst/tests/test_rdd.R

```diff
@@ -643,3 +643,8 @@ test_that("collectAsMap() on a pairwise RDD", {
   vals <- collectAsMap(rdd)
   expect_equal(vals, list(`1` = "a", `2` = "b"))
 })
+
+test_that("show()", {
+  rdd <- parallelize(sc, list())    
```

Comment:
Could you pass in `1:10` or `nums` (defined above) to parallelize instead of an empty list ?